imports/ui/report-wizard/report-wizard.jsx

```diff
@@ -97,7 +112,39 @@ class ReportWizard extends Component {
         <InnerAppBar onBack={() => dispatch(goBack())} title={reportItem.title} />
         <div className='flex-grow bg-white flex flex-column overflow-auto pa3'>
           <div>
-            {infoItemMembers('Report title', reportItem.title)}
+            { isEditable ? (
+              <div>
+                <form onSubmit={this.handleSubmit}>
+                  <div className='relative'>
+                    <div className='mt1 f6 bondi-blue'>Edit title</div>
+                    <EditableItem
+                      label=''
+                      name={reportItem.title}
+                      key={reportItem.title}
+                      initialValue={reportItem.title}
+                      onEdit={val => this.setState({reportTitle: val})}
+                    />
+                    <div className='absolute right-0 tl f6 bondi-blue fw5'>
+                      <FlatButton onClick={() => this.setState({isEditable: false})} style={{minWidth: '50px'}}>
+                        <span className='silver'> Cancel</span>
+                      </FlatButton>
+                      <FlatButton type='submit' style={{minWidth: '50px'}}>
+                        <span className={(reportTitle ? 'bondi-blue' : 'silver')} >
```

Comment:
Sorry @sophieyang0813 , just one last thing I just noticed.
The button changes color to appear "disabled", but it's not really disabled, it'll still work as normal if the title is empty.
You can disable it completely with the button's "disabled" attribute, or check `if (!reportTitle) return` at the start of `handleSubmit` to truly prevent a save attempt of an empty title